### DISCUSSION TITLE
Simple-broker without wifi?

### DISCUSSION THREAD

**Discussion by yltsa** [hsaturn/TinyMqtt#67] [Q&A]
Hi,

is it possible to modify the simple-broker example working without Wifi? I have an Ethernet in Esp32 and would like to make a small MQTT server.

BR, Ilkka

**Comment by real-bombinho** [comment]
It is absolutely possible and I did it already in my fork. This is on my todo list to discuss the inclusion of Ethernet and some hurdles have already gotten fixed recently in the main branch. 

edit: @hsaturn has already implemented it, thank you.
